.vscode/settings.json

```diff
@@ -21,4 +21,4 @@
     "python.analysis.extraPaths": [
         "./src"
     ],
-}
+}
```

Comment:
I would remove this: you can do ctrl+alt+P in vscode and type in 'save without formatting' to stop it from adding a new line

Reply:
I think I'll just leave it, I'd rather just bow to the formatting gods